uavcan/equipment/air_data/1027.RawAirData.uavcan

```diff
@@ -0,0 +1,41 @@
+#
+# Raw Air Data.
+#
+
+# Note: unused vars should be assigned NaN
```

Comment:
@pavel-kirienko I don't see such data coming from the same sensor. But, same module sending both is entirely possible. My initial PR was for only one kind of raw data i.e. AirSpeed related, as my module only sends that data. I do like the idea of splitting them up. I do see a possibility of correlation b/w static and diff pressure. Although, not sure if that's going to be used or even be needed.